### PR TITLE
Add overlay for Redmi7A

### PR DESCRIPTION
**Pull request by YaAlex3**

### CODE REVIEW

File: Xiaomi/Redmi7A/res/values/config.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <integer name="config_screenBrightnessDoze">17</integer>
+    <integer name="config_screenBrightnessSettingDefault">536</integer>
+    <integer name="config_screenBrightnessSettingMaximum">2048</integer>
+    <integer name="config_screenBrightnessSettingMinimum">7</integer>
+    <integer name="config_bluetooth_idle_cur_ma">0</integer>
+    <integer name="config_bluetooth_operating_voltage_mv">3300</integer>
+    <integer name="config_bluetooth_rx_cur_ma">0</integer>
+    <integer name="config_bluetooth_tx_cur_ma">0</integer>
+    <integer name="config_shutdownBatteryTemperature">600</integer>
+    <dimen name="rounded_corner_radius">0px</dimen>
```
Comment (phhusson) [inline Xiaomi/Redmi7A/res/values/config.xml]: if i'm not mistaken, this doesn't work, does it?
I seem to remember this must be in systemui not framework-res

**Comment by phhusson** [comment]
CI error is 
`Fatal: ./Xiaomi/Redmi6A/AndroidManifest.xml: priority 57 conflicts with another manifest`

**Comment by YaAlex3** [comment]
> CI error

Should be resolved now.

File: Xiaomi/Redmi7A/res/values/config.xml (hunk 1 above)
Comment (YaAlex3) [reply Xiaomi/Redmi7A/res/values/config.xml]: nah, i think i checked most of the values to be in framework res, and edited them to match 7A's values

**Pull request merged**